MapleServer2/PacketHandlers/Game/RequestTaxiHandler.cs

```diff
@@ -17,21 +17,32 @@ class RequestTaxiHandler : GamePacketHandler
         public override void Handle(GameSession session, PacketReader packet)
         {
             byte mode = packet.ReadByte();
-            int mapId = packet.ReadInt();
 
             switch (mode)
             {
-                case 3: // rotors using mesos?
+                case 3: // rotors using mesos
+                    session.Player.Mesos -= 60000;
+                    session.Send(MesosPacket.UpdateMesos(session));
+                    break;
+                case 4: // rotors using merets
+                    session.Player.Merets -= 15;
+                    session.Send(MeretsPacket.UpdateMerets(session));
                     break;
+                case 5: // is sent after using rotors with meret, idk why..
+                    return;
             }
 
-            //TODO: figure out when to pay which currency Merits/Mesos?
+            int mapId = packet.ReadInt();
+
+            System.Console.WriteLine(mapId);
```

Comment:
Instead of using the `System.Console` use the Logger that is provide in each Handler `logger.Debug($"{mapId}");`

Reply:
i removed it, since i used it only for debugging